src/Inventory/Asset/Cartridge.php

```diff
@@ -41,7 +41,7 @@
 
 class Cartridge extends InventoryAsset
 {
-    public function knownTags(): array
+    public function knownTags(bool $withstates = true): array
```

Comment:
Can you explain this change ?

Reply:
I changed this function in order to reuse it for the cartridge type mapping to the type inventory process return. As originally it was creating Max, Used and Remainng instances for each type and this information is not usefull for the second usage I introduced, I just added a parameter defaulting to the previous behaviour suppressing it.